code/modules/awaymissions/mission_code/ghost_role_spawners/golems.dm

```diff
@@ -134,6 +134,24 @@
 		user.death()
 		return
 
+/obj/effect/mob_spawn/human/golem/attackby(obj/item/slimepotion/transference/I, mob/user, params)
+	. = ..()
+	if(istype(src, /obj/effect/mob_spawn/human/golem/servant))
+		src.has_owner = FALSE
+	src.flavour_text = null
+	if(iscarbon(user) && can_transfer)
+		var/human_transfer_choice = alert("Transfer your soul to [src]? (Warning, your old body will die!)",,"Yes","No")
```

Comment:
*Maybe* change the `title` argument to `null` here? Doesn't actually make a difference in the end but it's better for readability.
```suggestion
		var/human_transfer_choice = alert("Transfer your soul to [src]? (Warning, your old body will die!)", null,"Yes","No")
```